packages/common-common/src/cacheKeyUtils.ts

```diff
@@ -1,5 +1,21 @@
 import { Request } from 'express';
 
+export type CacheKeyDuration = {
+  cacheKey: string;
+  cacheDuration: number;
+};
+
+// Extend the Request type with your custom properties
+export interface CustomRequest extends Request, CacheKeyDuration {}
```

Comment:
Why do we need a CustomRequest type? Will this play nice with TypedRequest / TypedResponse?

Also, we already use the global namespace to augment express requests in the `commonwealth/server/types.ts` file. Why not add the changes there?

Reply:
Typescript was complaining about - `req.cacheKey` and `req.cacheDuration` 
- it just attach two optional properties.
- its working fine - no linting/runtime errors due to this!